src/ImageSharp/Formats/ImageExtensions.Save.tt

```diff
@@ -17,6 +17,7 @@ using SixLabors.ImageSharp.Advanced;
         "Jpeg",
         "Png",
         "Tga",
+        "Tiff",
```

Comment:
I suggest to add something like this in the `foreach (string fmt in formats)` loop:

```C#
string experimentalString =  fmt != "Tiff" ? "" : @"EXPERIMENTAL! ";
#>

        /// <summary>
        /// <#= experimentalString #>Saves the image to the given stream with the <#= fmt #> format.	        /// Saves the image to the given stream with the <#= fmt #> format.
        /// </summary>

// same in other overloads
```

Reply:
done